### ISSUE TITLE
Error handling for wrong formatted arguments

### ISSUE THREAD

**Issue by dennismckinnon** [eris-ltd/eris-contracts.js#2]
Example code

```
function test_get(req, res, next){
	if ((req.query.shiftnum != undefined)){
		console.log("happy happy happy!")
	} else {
		console.log("So very very very sad :(")
	}
	contract.shiftExist.call(req.query.shiftnum, function(error, output){
		if (error) {
			console.log("error occured")
			res.writeHeader(500, {"Content-Type": "text/plain"})
			res.write("An error occurred in the blockchain")
			res.end()

		} else {
			console.log("no error")
			res.json(output)
		}
	});
	next()
}
```

the shiftExist takes a uint as an argument. If i call with a string I get the error:

{"code":"InternalError","message":"new BigNumber() not a number: hello"}

and the callback function is not called. Ideally The callback should be called with a non-empty error so it can be handled gracefully.

**Comment by androlo** [comment]
I'll add a try-catch around the formatting for now. It'll be in the next push. We'll add something more elegant later.

That should solve big number stuff and other common things as well. You'd get an error in the callback.

**Comment by androlo** [comment]
implemented in 0.10.14 (i believe?). Anyways, it's in. Gonna go into more advanced type checking as well - leaving this open.
